### DISCUSSION TITLE
CCAve Add on not working with elementor Plugin

### DISCUSSION THREAD

**Discussion by itops-ccmt** [impress-org/givewp#5848] [Developer Q&A] [answered]
Hi

I'm using elementor 3.2.2 and I notice that CCAve Add-on doesn't work when the elementor plugin is activated.

Issue: When user selects CCAVe as payment method and submits the donation form, user is not redirected to CCAve payment page instead website home page is opened.

Fix. Add some dummy page name instead of root in pugins\give-ccavenue\includes\donation-processing.php.

**Existing code** :
wp_safe_redirect( home_url( "/?process_ccavenue_payment=processing&donation={$donation_id}&form-id={$form_id}" ) );

**Modified code:**
wp_safe_redirect( home_url( "/ccave_payment?process_ccavenue_payment=processing&donation={$donation_id}&form-id={$form_id}" ) );

Please review this code and include in your patch or suggest some alternative fix.

**Comment by JasonTheAdams** [comment, marked as answer]
Hi @itops-ccmt!

This type of thing is considered Product Support, which is not the [intended use of Discussions](https://github.com/impress-org/givewp/discussions/5816). The best way to get help with this is to [reach out to our awesome support team](https://givewp.com/support) so they can look into it further and make a ticket for a fix.

Thanks for reaching out!

**Comment by itops-ccmt** [comment]
It was on the request of the support team I had posted it here. Below is
the mail from your support team

*Hasan* replied
May 27, 4:48am
Hi Suresh,

For anything development delated, kindly post your queries here directly to
our developers https://github.com/impress-org/givewp/discussions and read
this https://github.com/impress-org/givewp/discussions/5816

They'll give you more detailed insights.



On Thu, May 27, 2021 at 11:47 PM 'Jason Adams' via CCMT IT Support <
***@***.***> wrote:

> Hi @itops-ccmt <https://github.com/itops-ccmt>!
>
> This type of thing is considered Product Support, which is not the intended
> use of Discussions
> <https://github.com/impress-org/givewp/discussions/5816>. The best way to
> get help with this is to reach out to our awesome support team
> <https://givewp.com/support> so they can look into it further and make a
> ticket for a fix.
>
> Thanks for reaching out!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/impress-org/givewp/discussions/5848#discussioncomment-794560>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQNBYXMBDRKTERF6WZOEVTLTP2EETANCNFSM45UFK7OQ>
> .
>
-- 
*Suresh Bathala*
Chief Technology Officer | Central Chinmaya Mission Trust
Saki Vihar Road, Powai, Mumbai 400 072 | (M): +91 805 608 7838

**Comment by Benunc** [comment]
Hey @itops-ccmt that's our bad as a support team. The discussions here are more for folks building integrations, not for bug reports. I am testing this and will reply to your support ticket soon. Thanks!
